### PR TITLE
Change navbar and modal theming

### PR DESCRIPTION
**Pull request by Var1377**
Navbar is now white by popular request

(Changing the MUI primary colour also improves the look of other components as it was closer to what it was developed with)

### CODE REVIEW

**Review by DanWilkins2107 — APPROVED** [review]

File: src/components/Buttons/LinkButton.tsx [hunk 1]
```diff
@@ -10,10 +10,6 @@ const UnstyledLink = styled(Link)`
     display: contents;
 `;
 
-const UpperCaseButton = styled(Button)`
-    text-transform: uppercase;
```
Comment (TomCusack-Softwire) [inline src/components/Buttons/LinkButton.tsx:14]: why is this deleted? I thought people wanted it uppercase (even if it doesn't _yet_ work on `prod` build!)

**Review by TomCusack-Softwire — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -37,6 +37,9 @@ next-env.d.ts
 # webstorm
 .idea
 
+# vscode
+.vscode
```
Comment (DanWilkins2107) [inline .gitignore:41]: What acc is this?

File: src/components/Buttons/LinkButton.tsx (hunk 1 above)
Comment (DanWilkins2107) [reply src/components/Buttons/LinkButton.tsx:14]: Obviously correct me if I'm wrong on this, why couldn't we just type in uppercase if we want the button to be uppercase instead of having a separate component?

**Review by DanWilkins2107 — APPROVED** [review]
Couple Qs from me, nothing to change though I don't think
